.clang-format

```diff
@@ -1,47 +1,63 @@
-AccessModifierOffset: -3
-AlignEscapedNewlinesLeft: true
-AlignTrailingComments: true
-AllowAllParametersOfDeclarationOnNextLine: false
-AllowShortFunctionsOnASingleLine: true
-AllowShortIfStatementsOnASingleLine: false
-AllowShortLoopsOnASingleLine: false
-AlwaysBreakBeforeMultilineStrings: false
-AlwaysBreakTemplateDeclarations: true
-BinPackParameters: false
-BreakBeforeBinaryOperators: false
+---
+AccessModifierOffset: '-3'
```

Comment:
Presumably this gives us the 1 space indent for the access modifiers. I think we should switch this to -4 and reformat the code.